ui/pages/swaps/awaiting-swap/awaiting-swap.js

```diff
@@ -138,10 +138,11 @@ export default function AwaitingSwap({
 
   const baseNetworkUrl =
     rpcPrefs.blockExplorerUrl ??
-    SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP[chainId] ??
+    SWAPS_CHAINID_DEFAULT_BLOCK_EXPLORER_URL_MAP[caipChainId] ??
     null;
   const blockExplorerUrl = getBlockExplorerLink(
-    { hash: txHash, chainId },
+    // does this need to be updated?
+    { hash: txHash, caipChainId },
```

Comment:
need to check `@metamask/etherscan-link` or just convert it back to chainId here